modules/User/Controllers/UserController.php

```diff
@@ -46,6 +46,11 @@ public function show(GetUserRequest $request): JsonResponse
         return Json::buildItems('user', $presenter->getData());
     }
 
+    public function me()
+    {
+        return Json::buildItems('user', (new UserPresenter(auth()->user()))->getData(),"",200);
```

Comment:
split the presenter please 
```suggestion
        $user = auth()->user();
        $userPresenter = new UserPresenter($user);
        return Json::buildItems('user', $userPresenter)->getData(),"",200);
```